eventhandling/getSliEvent.go

```diff
@@ -47,33 +47,82 @@ func (eh GetSliEventHandler) HandleEvent() error {
 
 	// send started event
 	_, err = eh.keptnHandler.SendTaskStartedEvent(eventData, utils.ServiceName)
-
 	if err != nil {
-		errMsg := fmt.Sprintf("Failed to send task started CloudEvent (%s), aborting...", err.Error())
-		log.Println(errMsg)
+		errMsg := fmt.Errorf("failed to send task started CloudEvent: %w", err)
+		log.Println(errMsg.Error())
 		return err
```

Comment:
Please return the wrapping error...
furthermore if an error is returned, no need to log it, the code that will handle the error will decide if logging it or doing something else with it

```suggestion
		return fmt.Errorf("failed to send task started CloudEvent: %w", err)
```